tests/utils/user-instructions.ts

```diff
@@ -111,3 +111,63 @@ export const borrowIx = (program: Program<Marginfi>, args: BorrowIxArgs) => {
 
   return ix;
 };
+
+export type LiquidateIxArgs = {
+  marginfiGroup: PublicKey;
+  assetBankKey: PublicKey;
+  liabilityBankKey: PublicKey;
+  liquidatorMarginfiAccount: PublicKey;
+  liquidatorMarginfiAccountAuthority: PublicKey;
+  liquidateeMarginfiAccount: PublicKey;
+  bankLiquidityVaultAuthority: PublicKey;
+  bankLiquidityVault: PublicKey;
```

Comment:
Nit: I think we can derive the vaults and authorities inside the function so they don't have to be passed as args. 

We should update the constraints in the liquidate function to `has_one` instead of `constraint = ....`, this will automatically imply the arguments so they don't have to be passed.